sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/documentanalysis/models/DocumentField.java

```diff
@@ -22,6 +23,7 @@ public final class DocumentField extends TypedDocumentField<Object> {
      *
      * @return the value.
      */
+    @JsonIgnore
```

Comment:
Mind leaving a non-Javadoc comment somewhere in this class that any new getters will need to include JsonIgnore to prevent serialization and deserialization issues. Also, we should add a test to validate that serialization of this class and a `TypedDocumentField<Object>` with the same values are the same.